### PR TITLE
Add the updates without the high-level summary

### PR DESCRIPTION
**Pull request by dnadales**

### CODE REVIEW

**Review by jorisdral — APPROVED** [review]

File: blog/2023-02-22-consensus.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+benchmarks with a snapshotting policy that more closely resembles the one from
+mainnet.
+
+At the same time, we continued refactoring and cleaning up the prototype. 
+
+Also, we prepared the `anti-diff` packages (`fingertree-rm`, `diff-containers`,
+`simple-semigroupoids`) for publication in CHaP. This involved setting up CI,
+documentation, and some refactoring. These packages are now published to
+[CHaP][chap]. Also we published `cardano-lmdb` and `cardano-lmdb-simple` to
+CHaP.
```
Comment (Jasagredo) [inline blog/2023-02-22-consensus.md:29]:
```suggestion
Also, we published the `anti-diff` packages (`fingertree-rm`, `diff-containers`,
`simple-semigroupoids`) and the LMDB packages (renamed to 
`cardano-lmdb` and `cardano-lmdb-simple`)  in [CHaP][chap]. 
```

I saw too many times the word `CHaP`, Feel free to dismiss

File: blog/2023-02-22-consensus.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+feedback. Also, we improved the documentation related to followers
+([#4372][pull-4372]).
+
+We are also working on a design for optimizing the way we handle blocks from the
+future.
+
+### Support
+
+We released `ouroboros-consensus` 0.2.0.0 and ouroboros-consensus-cardano
+0.3.0.0 into CHaP.
```
Comment (Jasagredo) [inline blog/2023-02-22-consensus.md:59]:
```suggestion
We released `ouroboros-consensus` 0.2.0.0 and `ouroboros-consensus-cardano`
0.3.0.0 into CHaP. As a reminder, we split the packages related to Consensus
in two bundles, one with the core functionality, Cardano-agnostic code, and 
another bundle with the instantiations specific to Cardano.
```

Something like this?

File: blog/2023-02-22-consensus.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+title: Consensus Team Update
+slug: 2023-02-22-consensus
+authors: dnadales
+tags: [consensus]
+hide_table_of_contents: false
+---
+
+## High level summary
+
```
Comment (Jasagredo) [inline blog/2023-02-22-consensus.md:10]: This is still missing a high level summary? Or shall we just remove this section

**Review by Jasagredo — APPROVED** [review]

File: blog/2023-02-22-consensus.md (hunk 1 above)
Comment (dnadales) [reply blog/2023-02-22-consensus.md:29]: Good point. We should keep things simple, also in the prose :) I'll change this.

**Review by amesgen — APPROVED** [review]

File: blog/2023-02-22-consensus.md (hunk 3 above)
Comment (amesgen) [reply blog/2023-02-22-consensus.md:10]: On Slack, by @dnadales:

> (I'll add the high-level summary after I meet with Arnaud)

File: blog/2023-02-22-consensus.md [hunk 4]
```diff
@@ -8,6 +8,25 @@ hide_table_of_contents: false
 
 ## High level summary
 
+During the past two weeks we got the results from the system level benchmarks
+for UTxO HD. They showed a substantial performance regression, so we spent some
+time analyzing the results. We found out the frequency at which ledger snapshots
+were take was too high, so we requested the benchmarking team a new run with a
```
Comment (jorisdral) [inline blog/2023-02-22-consensus.md:14]:
```suggestion
were taken was too high, so we requested the benchmarking team a new run with a
```

File: blog/2023-02-22-consensus.md [hunk 5]
```diff
@@ -8,6 +8,25 @@ hide_table_of_contents: false
...
+more realistic snapshotting policy. We continued refactoring and improving the
+prototype, and we released UTxO HD related packages to CHaP.
```
Comment (Jasagredo) [inline blog/2023-02-22-consensus.md:16]:
```suggestion
prototype, and we released UTxO-HD related packages to CHaP.
```

File: blog/2023-02-22-consensus.md [hunk 6]
```diff
@@ -8,6 +8,25 @@ hide_table_of_contents: false
...
+
+We met with IOG researchers and networking specialists to discuss the Genesis
+design, which was well received. We continued working on testing and
+benchmarking different Genesis prototypes.
+
+We are also working on solving a test failure related to iterators. This work
+derived in several improvements such as better documentation, a framework for
+writing unit (and regression) tests, and the possibility to debugging
```
Comment (jorisdral) [inline blog/2023-02-22-consensus.md:24]:
```suggestion
writing unit (and regression) tests, and the possibility of/for debugging
```

**Review by Jasagredo — APPROVED** [review]

**Review by jorisdral — APPROVED** [review]

**Pull request merged**